metadata.yaml

```diff
@@ -19,14 +19,31 @@ tags:
     - monitoring
     - alerting
     - grafana
-series:
-    - kubernetes
+bases:
+    - name: ubuntu
+      channel: 20.04/stable
+containers:
+  prometheus:
+    resource: prometheus-image
+    mounts:
+      - storage: database
+        location: /var/lib/prometheus
+provides:
+  monitoring:
+    interface: prometheus
 requires:
     grafana-source:
         interface: grafana-datasource
     alertmanager:
         interface: alertmanager
+peers:
+  prometheus:
```

Comment:
I like the idea of naming the peer relation "replicas". I think that brings more clarity at first glance.

Reply:
One needs to be careful here. Peers are not always "replicas". For example in mongodb they can be members of a sharded group. Even though this feature is not currently supported by the charm, the fact that peers need not be replicas I would be hesitant to accept a blanket prescription that peer relations should always be named "replicas".

Reply:
Also true for Grafana, Alertmanager, and others. I worry that it's too application-specific to give it a blanket "replicas", and "peers" allows the charm author to decide what that should mean.

Reply:
In view of the above. I am leaving this as is. Resolving conversation for now. Feel free to reopen if there are any new concerns.

Reply:
I did not mean as a blanket rule: if your charm has only one type of peers then plain "replicas" could do. Otherwise you could be more explicit, e.g. "prometheus-replicas". We tend to name everything with the same string literal - service, container, interface, peer, ... all named "prometheus" ... from which it is a bit difficult to draw context at first glance. Anyway, this was just a suggestion.